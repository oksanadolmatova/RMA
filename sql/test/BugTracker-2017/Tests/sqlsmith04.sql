-- After 154K queries.

select
  ref_696.id as c0,
  ref_699.tag as c1
from
  sys.functions as ref_678
        left join sys._columns as ref_695
          right join tmp._columns as ref_696
            inner join sys.geometry_columns as ref_697
              left join sys.users as ref_698
              on (ref_697.f_geometry_column = ref_698.name )
            on (ref_696.storage = ref_698.name )
          on (ref_695.storage = ref_698.name )
        on (ref_678.language = ref_698.default_schema )
      left join sys.queue as ref_699
      on (ref_678.language = ref_699.progress )
    left join sys.columns as ref_700
    on (ref_698.fullname = ref_700.name )
where ref_700.id is not NULL
limit 77; 
