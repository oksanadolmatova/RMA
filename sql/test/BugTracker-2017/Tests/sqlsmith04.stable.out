stdout of test 'sqlsmith04` in directory 'sql/test/BugTracker-2017` itself:


# 17:13:51 >  
# 17:13:51 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=33167" "--set" "mapi_usock=/var/tmp/mtest-17366/.s.monetdb.33167" "--set" "monet_prompt=" "--forcemito" "--dbpath=/home/niels/scratch/monetdb/Linux-x86_64/var/MonetDB/mTests_sql_test_BugTracker-2017"
# 17:13:51 >  

# MonetDB 5 server v11.26.0
# This is an unreleased version
# Serving database 'mTests_sql_test_BugTracker-2017', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 128bit integers
# Found 7.332 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2017 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://localhost.nes.nl:33167/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-17366/.s.monetdb.33167
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded

Ready.

# 17:13:51 >  
# 17:13:51 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-17366" "--port=33167"
# 17:13:51 >  

#select
#  ref_696.id as c0,
#  ref_699.tag as c1
#from
#  sys.functions as ref_678
#        left join sys._columns as ref_695
#          right join tmp._columns as ref_696
#            inner join sys.geometry_columns as ref_697
#              left join sys.users as ref_698
#              on (ref_697.f_geometry_column = ref_698.name )
#            on (ref_696.storage = ref_698.name )
#          on (ref_695.storage = ref_698.name )
#        on (ref_678.language = ref_698.default_schema )
#      left join sys.queue as ref_699
% .L146,	.L150 # table_name
% c0,	c1 # name
% int,	oid # type
% 1,	3 # length

# 17:13:52 >  
# 17:13:52 >  "Done."
# 17:13:52 >  

